app/api/models/statistics.py

```diff
@@ -60,6 +80,11 @@ class Regulation(BaseModel):
         default=None
     )
 
+    @field_validator("enhancers", "promoters", "ctcf_count", "tfbs_count", "open_chromatin_count", mode="before")
+    def _convert_int_fields(cls, value: Union[str, int, None]) -> Optional[int]:
+        """Convert string inputs to integers for the specified fields."""
+        return _str_to_int(value)
+
```

Comment:
Does Pydantic itself not do type conversion to whatever type ?

Reply:
As of the current setup, it does complain, what I see in the logs:
```
1 validation error for GenomeStatistics                                                    │
│ regulation_stats.`regulation.stats.enhancer_count`                                                                                                          │
│   Input should be a valid integer, unable to parse string as an integer [type=int_parsing, input_value='70168\u2002', input_type=str]                       │
│     For further information visit https://errors.pydantic.dev/2.4/v/int_parsing
```

Reply:
This is what comes out from gRPC
```
{
  "genome_uuid": "8c31359e-d624-4635-860e-9a92cde3dac2",
  "statistics": [
    {
      "name": "regulation.stats.enhancer_count",
      "label": "regulation.enhancer_count",
      "statistic_type": "integer",
      "statistic_value": "70168 "  <-- /!\ there is space in this value
    },
    ...
  ]
}
```
However in the DB I saw and `ENSP` at the end
![image](https://github.com/user-attachments/assets/c9146770-d2ef-46f4-88af-c092c9653766)

> It's now fixed, but I took that screen shot before 😈

Reply:
Yeah, this PR is not going to help with letters in `int` field.